### DISCUSSION TITLE
Page Kinds?

### DISCUSSION THREAD

**Discussion by Wysardry** [getnikola/nikola#3691] [Q&A]
In the theming documentation, it mentions that there are several page kinds provided by default plugins. I can guess what most of them are for based on their names, but there are two that I'm unsure of.

What are _story_page_ and _generic_page_ used for and how are they treated differently to _post_page_ and _page_page_?

**Comment by Kwpolska** [comment]
`story_page` is the same as `page_page` (they always appear together). `story` is a legacy term for `page` that we still use internally.

`generic_page` can’t actually appear in a site that uses the default plugins, and it doesn’t seem to be able to appear in a site using any plugins in our repository either, so I removed it from the docs. (It would be used if someone used the generic renderer for Post objects, outside of the standard rendering of the timeline, and did not provide a pagekind — an unlikely situation overall).
